manual/manual/cmds/lexyacc.etex

```diff
@@ -52,8 +52,8 @@ The following command-line options are recognized by "ocamllex".
 \item["-ml"]
 Output code that does not use OCaml's built-in automata
 interpreter. Instead, the automaton is encoded by OCaml functions.
-This option mainly is useful for debugging "ocamllex", using it for
-production lexers is not recommended.
+This option improves performances when using the native compiler, but
```

Comment:
"performance" should always be singular in this context (common mistake)!  (Likewise "then" -> "it" later in the sentence.)

Reply:
Thanks!